backend/src/utils/graph_db_utils.py

```diff
@@ -25,3 +25,23 @@ def test_connection():
     finally:
         driver.close()
         return connection_healthy
+
+
+def create_query(llm_query):
```

Comment:
I wonder if we need to change the name of this function to `execute_query`. It looks like the query has already been created and this is just running it.

Reply:
yes I agree, I'll edit it